internal/controller/apiextensions/claim/reconciler.go

```diff
@@ -554,7 +554,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 		return reconcile.Result{}, errors.Wrap(r.client.Status().Update(ctx, cm, client.FieldOwner(fieldOwnerName)), errUpdateClaimStatus)
 	}
 
-	record.Event(cm, event.Normal(reasonBind, "Successfully bound composite resource"))
+	record.Event(cm, event.Normal(reasonBind, "Composite resource is ready"))
```

Comment:
What if the composite resource already existed for a while, then was later claimed? (i.e. Bound to a claim.)

The resource would have already been ready for some time, making this a bit misleading.